source/languageHandler.py

```diff
@@ -214,6 +214,7 @@ def setLocale(localeName: str) -> None:
 	try:
 		locale.setlocale(locale.LC_ALL, localeName)
 		locale.getlocale()
+		log.debug(f"setting python locale to {localeName}")

```

Comment:
I think this might be more accurate as "set python locale...", since it has already happened and succeeded. What do you think?